metarecord/views/function.py

```diff
@@ -255,11 +255,12 @@ def get_version_history(self, obj):
 class FunctionFilterSet(django_filters.FilterSet):
     class Meta:
         model = Function
-        fields = ('valid_at', 'version')
+        fields = ('valid_at', 'version', 'classification__code', 'information_system')
```

Comment:
Because there is field `classification_code` in Function API data, IMO it would be more consistent if the filter was also named `classification_code`.